pontoon/settings/base.py

```diff
@@ -28,11 +28,13 @@ def path(*args):
 # Is this a dev instance?
 DEV = os.environ.get('DJANGO_DEV', 'False') != 'False'
 
+MAINTAINER = os.environ.get('MAINTAINER', 'False') != 'False'
```

Comment:
Could you move this setting to the bottom of the file, where we keep Pontoon app-specific variables?

I'd give the variable a more self-explanatory name (`ENABLE_BUGS_TAB`?) and add a comment in the line above it:
```python
# Enable Bugs tab on team pages, pulling team data from bugzilla.mozilla.org.
# See bug 1567402 for details. A Mozilla-specific variable.
ENABLE_BUGS_TAB = os.environ.get('ENABLE_BUGS_TAB', 'False') != 'False'
```

We also need to [document](https://github.com/mozilla/pontoon/blob/248facc10aef3e41c1157d0afcbb3549432fc88f/docs/admin/deployment.rst#environment-variables) the variable (note that variables are listed in alphabetic order there).

Reply:
Within the documentation, do you want me to specify that it is a Mozilla specific variable?

Reply:
Yeah, the docs should basically says what's written in the comment (you can leave out the reference to the bug).